### ISSUE TITLE
Graphiques fiches actions-plus

### ISSUE THREAD

**Issue by Clemog** [datagir/nosgestesclimat#1423]
Aujourd'hui, suite à la migration du site ABC, certaines images/figures ne sont plus affichées.

C'est l'occasion de discuter du format de ces graphes, non accessibles et peu évolués (capture d'écran).

**Comment by Benjamin-Boisserie-ABC** [comment]
@Clemog J'ai récupéré les images. Je peux les mettre sous Github et reconstituer les liens mais quid en effet du format qui sont des simples captures d'écran (peu esthétique donc). Ne pourrait-on pas envisager une représentation graphique inspirée de l'esthétique du site ou des graphes présents dans votre article de [blog](https://datagir.ademe.fr/blog/budget-empreinte-carbone-c-est-quoi/) (type "dessin à main levée)

**Comment by Clemog** [comment]
Effectivement, je pense qu'il faudrait les reprendre mais dans tous les cas, il me semble important de ne pas laisser des liens cassés sur le site et dans l'attente de prioriser ce sujet, je pense qu'il faudrait les mettre dans le dépôt et récrire les liens

**Comment by laem** [comment]
Oui ce serait à mon avis mieux de faire des graphiques HTML/JS, plutôt que de refaire les images, ou même les reuploader. 

Pour les graphiques style main, oui c'est possible, voir cette bibliothèque https://github.com/timqian/chart.xkcd


![image](https://user-images.githubusercontent.com/1177762/178522927-e93f834f-31b2-4531-8ae7-14ad689d4996.png)

Mais ça demande du temps pour tout reprendre.
